web_services/foxx/brightid/db.js

```diff
@@ -600,15 +614,19 @@ function linkContextId(id, context, contextId, timestamp) {
     }
   }
 
-  if (getUserByContextId(coll, contextId)) {
-    throw 'contextId is duplicate';
-  }
-
   coll.insert({
     user: id,
     contextId,
     timestamp
   });
+
+  // sponsor the user if contextId is temporarily sponsored
+  const tempSponsorship = sponsorshipsColl.firstExample({ contextId });
+  if (tempSponsorship) {
+    const app = tempSponsorship._to.replace('apps/', '');
+    const operations = require('./operations');
```

Comment:
Why are we requiring `operations` inside a function that could be called many times with local scope so it gets thrown away?